youtube/feed.go

```diff
@@ -279,13 +279,74 @@ var (
 	ErrMaxCustomMessageLength = errors.New("max length of custom message can be 500 chars")
 )
 
-func (p *Plugin) parseYtUrl(channelUrl *url.URL) (idType ytUrlType, id string, err error) {
+var listParts = []string{"snippet"}
```

Comment:
This wasn't so much of a question of readability in this case, but rather, having a singular source of truth... this ensures the slice provided, which should be consistent between methods, is, so there aren't any errors caused by issues such as `"snippet"` being misspelt...

This is something of a question of preference for me though, and I wouldn't take issue with inlining it if that was considered preferable...

Reply:
Misspellings would be caught after even the lightest testing, so I'm personally unconvinced of the benefit. That's just my opinion, though; let's wait to see what Des thinks.

Reply:
Part of the issue in this case, is that it's a fetching error, which isn't propagated to the end user with exhaustive detail, so the issue would only show up in log errors, which is fine if you're running an instance of the bot, and have access to them, but not with access just to the control panel...

Reply:
Sorry, I don't follow -- how do end users play into this? To reiterate, my argument is that if this variable is meant to prevent typos, the benefit is minimal as any such typos would be readily noticed during development (where one would have access to logs).

Reply:
I think I got the wrong idea here... the thing is, as a single source of truth, you know if you test one of them, and it's wrong due to that variable being wrong, it'll be wrong for all of them, so when you fix it, it will fix it for all of them... it also means if it's ever broken, or can be updated with a better option in future, due to an API change, only this single source needs to be updated... got my ideas wrong there...